apps/server/src/modules/board/service/content-element-update.visitor.spec.ts

```diff
@@ -56,6 +58,13 @@ describe(ContentElementUpdateVisitor.name, () => {
 				await expect(() => updater.visitSubmissionItemAsync(submissionItem)).rejects.toThrow();
 			});
 		});
+
+		describe('when component is a drawing-item', () => {
+			it('should throw an error', async () => {
```

Comment:
from the code there is also a happy path. I dont see any tests for that

Reply:
Sorry, seeing constructions of tests in this file - this test should not be here.
Instead, by following the pattern, I will create test describe('when visiting a drawing element ...